### PR TITLE
Add debug logging management to config info

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

This is a first pass of https://github.com/home-assistant/architecture/discussions/715

The goal of this change is to make it easy to turn the debug logging on for collecting something you need to wait a while for it to happen.  Ie your device breaks at 3am every day.

For actions that can repro a problem right away, something like what is suggested here https://github.com/home-assistant/architecture/discussions/715#discussioncomment-2046206  and here https://github.com/home-assistant/architecture/discussions/715#discussioncomment-2064896 can be done in a second pass.

Requires https://github.com/home-assistant/core/pull/65158

UwwYvu5FSm

Narrow
Screen Shot 2022-01-28 at 18 15 25

Normal
Screen Shot 2022-01-28 at 18 15 30

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by bdraco** [comment]
I haven't gotten back to this because I was busy with beta.  Hopefully next week after .1

**Comment by frenck** [comment]
❤️ 

We should make sure it is added to the integration card as well, in here:

image

As download diagnostics and links to the issue trackers are there as well.

For the future, we could make a submenu in there maybe? With the possibility to download logs or view filtered logs specifically for this integration even (future tunes).

**Comment by bramkragten** [comment]
> filtered logs
@frenck https://github.com/home-assistant/frontend/pull/11166

File: src/panels/config/info/integrations-card.ts [hunk 1]
```diff
@@ -128,10 +152,27 @@ class IntegrationsCard extends LitElement {
                       : html`
                           <td>${docLink}</td>
                           <td>${issueLink}</td>
-                          <td class="setup">
-                            ${setupSeconds ? html`${setupSeconds} s` : ""}
-                          </td>
                         `}
+                    <td class="log">
+                      ${logLevel === 10
+                        ? html`<mwc-button
```
Comment (bramkragten) [inline src/panels/config/info/integrations-card.ts:158]: Can we make this a toggle instead of a button?

Comment (bdraco) [reply src/panels/config/info/integrations-card.ts:158]: I'm going to reset and start over with a new design after everything is mopped up from the 2022.2 release.  I'll send some mock ups once I have cycled back to this. 

Thanks!

**Comment by bdraco** [comment]
This one needs to be rethought

**Pull request closed without merging**